README.md

```diff
@@ -109,6 +109,7 @@ An options-object with the following attributes:
 | `flipDurationMs` | Number | No | `0` | The same value you give the flip animation on the items (to make them animated as they "make space" for the dragged item). Set to zero or leave out if you don't want animations |
 | `type` | String | No | Internal | dnd-zones that share the same type can have elements from one dragged into another. By default, all dnd-zones have the same type |
 | `dragDisabled` | Boolean | No | `false` | Setting it to true will make it impossible to drag elements out of the dnd-zone. You can change it at any time, and the zone will adjust on the fly |
+| `morphDisabled` | Boolean | No | `false` | Dragged item is morphed to target zone dimensions by default. You can prevent this by setting this option. |
```

Comment:
Maybe we can make this a bit more accurate. something like:
`By default, when dragging over a zone, the dragged element is morphed to look like it would if dropped. You can prevent it by setting this option.`
The current explanation suggests that only the dimensions change but in fact a lot of styles are copied. 
It also suggest it is morphed to the target zone but in fact it is target index within the zone and doesn't happen when it would go back to its original place (when dragged outsize of any zone).

Reply:
I do aggree, thanks for explanation.